specification/cognitiveservices/data-plane/UnifiedVision/preview/2023-07-01-preview/UnifiedVision.json

```diff
@@ -0,0 +1,3462 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Computer Vision Client",
+    "description": "The Computer Vision API provides state-of-the-art algorithms to process images and return information. For example, it can be used to extract text using Read OCR, caption an image using descriptive natural language, detect objects, people, and more.",
+    "version": "2023-07-01-preview"
+  },
+  "paths": {
+    "/imageanalysis:segment": {
```

Comment:
If you use the POST action pattern, the action has to be a verb. Seems "/imagesanalysis:analyze" makes more sense given your description.

Reply:
I see this is an old endpoint, but my concern still stands but is moot now. Please consider for future services.